wasp/src/main/java/com/orhanobut/wasp/WaspResponse.java

```diff
@@ -54,4 +54,8 @@ public void log(LogLevel logLevel) {
                 // Method is called but log level is not meant to log anything
         }
     }
+
+    public WaspError toWaspError(String errorMsg) {
```

Comment:
instead of errorMsg, we can just say <code>error</code>, we don't use abbreviation.